app/routes/bdb/components/BdbDetail.js

```diff
@@ -226,6 +229,22 @@ export default class BdbDetail extends Component<Props, State> {
           </tr>
         ));
 
+    const surveys =
+      companySurveys &&
+      companySurveys.slice(0, this.state.surveysToDisplay).map(survey => (
+        <tr key={survey.id}>
+          <td>
+            <Link to={`/surveys/${survey.id}`}>{survey.title}</Link>
+          </td>
+          <td>{eventTypes[survey.event.eventType]}</td>
+          <td>
+            <Time time={survey.activeFrom} format="DD.MM.YYYY" />
+          </td>
+          <td>{truncateString(survey.event.location, 50)}</td>
+          <td>{truncateString(survey.event.description, 70)}</td>
+        </tr>
```

Comment:
Let's remove this and just have a link to the survey for each event in that list, at least for now.

Reply:
Agreed :smile: